desktop/src/main/java/bisq/desktop/main/dao/bonding/roles/RolesView.java

```diff
@@ -157,6 +157,9 @@ public void updateItem(final RolesListItem item, boolean empty) {
                         };
                     }
                 });
+        column.setComparator(Comparator.comparing(RolesListItem::getLockupDate).reversed());
+        column.setSortType(TableColumn.SortType.ASCENDING);
+        tableView.getSortOrder().add(column);
```

Comment:
Why are you sorting this based on lockup date if the name is displayed without any date information displayed. This just makes it confusing for the user.